client/packages/common/src/ui/layout/tables/material-react-table/useBaseMaterialTable.tsx

```diff
@@ -103,9 +104,15 @@ export const useBaseMaterialTable = <T extends MRT_RowData>({
     muiTablePaperProps: {
       sx: { width: '100%', display: 'flex', flexDirection: 'column' },
     },
-    muiTableHeadCellProps: ({ column }) => ({
+    muiTableProps: {
+      // Need to apply this here so that relative sizes (ems, %) within table
+      // are correct
+      sx: theme => ({ fontSize: theme.typography.body1.fontSize }),
+    },
+    muiTableHeadCellProps: ({ column, table }) => ({
       sx: {
         fontWeight: 600,
+        fontSize: table.getState().density === 'compact' ? '0.95em' : '1em',
```

Comment:
```suggestion
```
font size for headers was 12px in designs, and I think this is a good call esp for longer column names in french etc.